basic-map.ipynb

```diff
@@ -0,0 +1,82 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Import the PyGMT library\n",
+    "import pygmt\n",
+    "\n",
+    "region=[127.5, 128.5, 26, 27]\n",
+    "projection=\"M10c\"\n",
+    "\n",
+    "\n",
+    "# Use the Figure class to create a figure object, named \"fig\"\n",
+    "fig = pygmt.Figure()\n",
+    "fig.coast(\n",
+    "    region=region,\n",
+    "    projection=projection,\n",
+    "    # Use default setting for figure's frame\n",
+    "    frame=True,\n",
```

Comment:
Hm. I am currently wondering, whether it is better to use here `pygmt.Figure.basemap` for the `region`, `projection`, and `frame` parameters?

Reply:
Ah, now I get from the title of this PR that this example is explicitly set up to show the `pygmt.Figure.coast` method. So, just ignore this comment 🙂.